homeassistant/components/hydrawise/switch.py

```diff
@@ -52,9 +52,9 @@
         vol.Optional(CONF_MONITORED_CONDITIONS, default=SWITCH_KEYS): vol.All(
             cv.ensure_list, [vol.In(SWITCH_KEYS)]
         ),
-        vol.Optional(CONF_WATERING_TIME, default=DEFAULT_WATERING_TIME): vol.All(
-            vol.In(ALLOWED_WATERING_TIME)
-        ),
+        vol.Optional(
+            CONF_WATERING_TIME, default=DEFAULT_WATERING_TIME.seconds
+        ): vol.All(vol.In(ALLOWED_WATERING_TIME)),
```

Comment:
This won't work as `ALLOWED_WATERING_TIME` is still a list of integers representing minutes. We shouldn't change the default or validation either as then it doesn't make sense anymore with the config option that is named `"watering_minutes"`. Convert the time to seconds internally after the config validation instead.

Reply:
Ah, whoops. I'll just convert it back to minutes here.